lib/Jsrt/JsrtInternal.h

```diff
@@ -14,6 +14,17 @@ typedef TTD::TTDJsRTActionResultAutoRecorder TTDRecorder;
 typedef struct {} TTDRecorder;
 #endif
 
+#ifndef NTBUILD // non-browser
+#define JSRT_UNSAFE_MODE
+#endif
+
+// JSRT Unsafe mode leaves runtime health-state responsibility to the host
+#ifdef JSRT_UNSAFE_MODE
+#define JSRT_MAYBE_TRUE false
+#else
+#define JSRT_MAYBE_TRUE true
```

Comment:
I think this name is confusing to say the least. Since it will be used [only] as the `verifyRuntimeState` parameter, I think should call it `JSRT_VERIFY_RUNTIME_STATE` or similar. This suggests the meaning of the value, rather than what the value might be. I can't imagine a "maybe-true" value controlled by boolean compile-time flag for `UNSAFE` being used in more a general sense.